csrc/lower_utils.cpp

```diff
@@ -556,7 +556,12 @@ class ReplaceExprInput : private kir::ExprMutator {
           replaced_inputs->at(node->inA()),
           replaced_inputs->at(node->inB()),
           node->init(),
-          node->options());
+          node->options(),
+          node->mAxes(),
+          node->nAxes(),
+          node->kAxes(),
+          node->batchAxes(),
+          node->inputLayout());
```

Comment:
This can potentially cause wrong result because for example I can replace the input tensor with something different to change this MMA from a TT to a TN, and this will lead to an inconsistency. But this problem was already there before this change, so I am fine with this change here.

Reply:
As you pointed out in the other comment, there is a risk of having invalid attributes if they are copied from parent `MmaOp` instance. I will revert API change and make sure that attributes are calculated at object construction.

Thanks for pointing this issue!